server/server.js

```diff
@@ -95,6 +96,7 @@ app.use(session({
 
 app.use(passport.initialize());
 app.use(passport.session());
+app.use('/api', requestsOfTypeJSON(), api);
```

Comment:
I think these public endpoints should be mounted at `/api` and all the rest that only the editor uses should be mounted at `/editor`?

Reply:
to me that sounds logical. do you think we should add the API versioning as well? i was trying to figure out what best practices are for this, but it seems like the options all have their own drawbacks (here's a [blog post](https://blog.restcase.com/restful-api-versioning-insights/) that breaks this down).

Reply:
I've added URL-based versioning in 57ced9c. Although it's not ideal, I think it's more obvious for casual users that may write scripts to import data into the Editor.

I'll change the namespaces of the private and public APIs in another PR.